src/index.ts

```diff
@@ -134,17 +140,26 @@ export class I2C extends Peripheral {
     super.destroy();
   }
 
-  private getDevice(address: number) {
+  private getDevice(address: number, busNumber?: number) {
     let device = this.devices[address];
 
     if (device === undefined) {
-      device = openSync(getBoardRevision() === VERSION_1_MODEL_B_REV_1 ? 0 : 1);
+      if (busNumber === undefined) {
+        busNumber = this.busNumber;
+      }
+      device = openSync(busNumber);
       this.devices[address] = device;
     }
 
     return device;
   }
 
+  public configure(options?: any): void {
```

Comment:
I'm sure it is working fine under your testing, but there is a chance for slight bugs given the right circumstances. This entire ecosystem is built around the assumption that the pins for any given instance of a peripheral will not change once instantiated. This is a hard rule that cannot be violated.